OMEdit/OMEditGUI/Options/OptionsDialog.cpp

```diff
@@ -4456,6 +4460,15 @@ FMIPage::FMIPage(OptionsDialog *pOptionsDialog)
   mpFMUNameLabel = new Label(tr("FMU Name:"));
   mpFMUNameTextBox = new QLineEdit;
   mpFMUNameTextBox->setPlaceholderText("<default>");
+  // Move FMU after build
+  mpMoveFMULabel = new Label(tr("Move FMU:"));
+  mpMoveFMUTextBox = new QLineEdit;
+  mpMoveFMUTextBox->setPlaceholderText(tr("<full file name with placeholders>"));
+  // placeholder count may change, don't invalidate translation
+  mpMoveFMUTextBox->setToolTip(tr("Placeholders: ") +
+                               FMIPage::FMU_FULL_CLASS_NAME_DOTS_PLACEHOLDER + ", " +
+                               FMIPage::FMU_FULL_CLASS_NAME_UNDERSCORES_PLACEHOLDER + ", " +
+                               FMIPage::FMU_SHORT_CLASS_NAME_PLACEHOLDER);
 #ifdef WIN32
```

Comment:
Its very annoying to type a full path manually. Add a browse button to select a path.
Also it should be possible to just specify a path without placeholders. Lets say I don't want to rename but just want to move the FMU to a specific location.

Reply:
Agree, this makes this feature work in a less surprising way.